.circleci/config.yml

```diff
@@ -44,10 +47,11 @@ jobs:
       - *RESTORE_REGISTRY
       - run: cargo generate-lockfile
       - *SAVE_REGISTRY
-      - run: rustc --version > ~/rust-version
-      - run: cargo test -p phf_macros
+      - run: rustc --version | tee ~/rust-version
+      - run: cargo test
+      - run: (cd phf_macros && cargo test --features unicase)
       - run: cargo test -p phf_codegen_test
-      - run: cargo test --manifest-path phf/Cargo.toml --features macros
+      - run: (cd phf_shared && cargo build --features core)
```

Comment:
Unrelated to this PR specifically, but we should switch this around to a default-enabled std feature.

Reply:
Yeah I was gonna ask about that; I always forget which way around it should be. I'll open an issue or I can do it in this PR.

Reply:
Separate PR probably.